service/src/main/scala/ch/epfl/bluebrain/nexus/admin/config/AdminConfig.scala

```diff
@@ -32,7 +30,7 @@ final case class AdminConfig(
 object AdminConfig {
 
   final case class ServiceAccountConfig(token: Option[String]) {
-    def credentials: Option[AuthToken] = token.map(AuthToken)
+    def credentials: Option[AccessToken] = token.map(AccessToken)
```

Comment:
It's a bit weird  to be  HTTP Bearer token when not doing interacting with a service via HTTP.  We can probably refactor that later.

Reply:
Yeah. The thing is we still a `Subject` to make calls to the `Realms` and `Acls` API.
We could have instead something like
```scala
final case class ServiceAccount(realm: String, name: String){
   def subject: Subject = User(realm, name)
}
```

But that's more prone to errors than having the actual saToken from where the subject is going to be extracted by the JWT automatically